src/test/resources/features/foi/CaseCreationStage.feature

```diff
@@ -0,0 +1,41 @@
+@FOICaseCreationStage @FOI @FOIWorkflow
+Feature: Case Creation Stage
+
+  Background:
+    Given I am logged into "CS" as user "FOI_USER"
+    When I create a "FOI" case and move it to the "Case Creation" stage
+    And I load the current case
+
+  #HOCS-3408
+  Scenario: User is able to complete the Case Creation stage
+    And I click the "Continue" button
+    And I select "Yes" for the validity of the request and continue
+    And I click the "Continue" button
```

Comment:
All 3 of these steps could be summed up in one step like 'And I submit that this is a valid request' or something. We want to take as many of the 'I click x' steps out as possible from now on

Reply:
The selecting validity step already has a click continue button in it, the one directly following it is for the screen after the validity selection screen